config/master_template.j2

```diff
@@ -0,0 +1,15 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{{ macros.header(imports=imports, ignore=ignore) }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases -%}
+    {{ test_case(case) }}
+    {% endfor %}
+
+    {% if additional_cases | length -%}
+    # Additional tests for this track
+    {% for case in additional_cases -%}
+    {{ test_case(case) }}
+    {% endfor %}
+    {%- endif %}
+{{ macros.footer(has_error_case) }}
```

Comment:
This is implicit now. For a template that extends this one, it should be possible to override this with `{%- set has_error_case = True -%}` if necessary.

```suggestion
{{ macros.footer() }}
```

Reply:
Commit reverted